__tests__/app.test.js

```diff
@@ -155,6 +155,43 @@ describe("/api/articles/:article_id", () => {
   });
 });
 
+describe("/api/articles", () => {
+  describe("GET", () => {
+    it("responds with an array of article objects each with appropriate properties", () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({body: {articles}}) => {
+        expect(Array.isArray(articles)).toBe(true)
+        expect(articles.length).toBeGreaterThan(0)
+        articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("number")
+        })
+      })
+    })
+    it("sorts the articles in date, descending order (newest to oldest)", () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({body: {articles}}) => {
+        console.log(articles[0])
+        const articleTimestamps = articles.map((article) => {
```

Comment:
Shouldn't need to separate out dates like this. Should be able to just test the objects with jest-sorted :)